### DISCUSSION TITLE
MAS n'est pas complet

### DISCUSSION THREAD

**Discussion by Aquaxium** [Monika-After-Story/MonikaModDev#9145] [QA - tech support + general] [answered]
j'ai regardé dans mes fichier et j'ai l'impression que j'ai pas de probleme mais quand je lance mon jeux je ne sais joué que a pong et je n'ai que 2 coupe de cheveux pour Monika . Est ce que quelqu'un pourrait m'aidé ?

**Comment by Wingdinggaster656** [comment, marked as answer]
Plus de fonctionnalités sont débloquées au fil du temps. Soyez patient.
